### DISCUSSION TITLE
Radio Köln

### DISCUSSION THREAD

**Discussion by Breuer1900** [junguler/m3u-radio-music-playlists#30] [General]
https://streams.rpr1.de/rpr-80er-128-mp3?usid=0-0-H-M-D-01

**Comment by junguler** [comment]
hi, thanks for opening this issue, do you want me to add this to a playlist? i presume the german or dutch file in the +merged+ folder?

**Comment by junguler** [comment]
this issue was converted to a discussion because it seems to be a better place for it
